kafkajobs/driver.go

```diff
@@ -501,3 +491,19 @@ func (d *Driver) requeueHandler() {
 		}
 	}
 }
+
+func pingKafka(client *kgo.Client, log *zap.Logger, timeout time.Duration, pipe jobs.Pipeline) error {
```

Comment:
No need to use name `pingKafka` 😃 Just `ping`, since we're inside the Kafka driver 😃